apps/mobile/src/components/splash-screen-guard/use-auth-context.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext, useContext } from 'react';
+
+interface AuthContextT {
+  lockApp(): void;
+}
+
+export const AuthContext = createContext<AuthContextT>({
```

Comment:
```suggestion
export const AuthContext = createContext<AuthContextArgs>({
```

not big on the T? That stands for type? Would avoid this in same way we don't use `IAuthContext` naming